POEM_104.md

```diff
@@ -0,0 +1,44 @@
+POEM ID: 104  
+Title:  Add PETSc Linear Solvers  
+authors: robfalck (Rob Falck), rob-hetterich (Rob Hetterich)  
+Competing POEMs: None  
+Related POEMs:  None  
+Associated implementation PR: [#3549](https://github.com/OpenMDAO/OpenMDAO/pull/3549)  
+
+Status:  
+
+- [ ] Active  
+- [x] Requesting decision  
+- [ ] Accepted  
+- [ ] Rejected  
+- [ ] Integrated  
+
+
+## Motivation
+
+The default implementation of DirectSolver, which uses Scipy's implementation of SuperLU is sometimes slower than other direct linear solvers available, notably umfpack and klu.
+
+
+
+## Description
+
+These linear solvers are available in PETSc, which is already a dependency of OpenMDAO. These solvers will be available via the new `PETScDirectSolver` class.
```

Comment:
For clarity..  PETSc is not currently a declared dependency (i.e. it is not installed vi the pyproject.toml).

The [documentation ](https://openmdao.org/swryan/newdocversions/latest/getting_started/getting_started.html?highlight=petsc) states:
```
Parallel Processing with MPI
OpenMDAO enables parallel processing via [MPI](https://www.mcs.anl.gov/research/projects/mpi/) and [PETSc](https://petsc.org/). This requires a few pre-requisites that can be installed with your package manager of choice: mpi4py and petsc4py.
```